src/dealDashboard/dealMenubar/dealMenubar.html

```diff
@@ -10,11 +10,14 @@
     <i class="fas fa-share-alt"></i>
     SHARE
   </a>
-  <a
-    if.bind="canEdit"
-    route-href="route.bind: deal.isOpenProposal ? 'editOpenProposal' : 'editPartneredDeal' ; params.bind: { id: deal.id, stageRoute: 'proposal' }">
-    <pbutton type="secondary">Edit deal</pbutton>
-  </a>
 
-  <pbutton click.delegate="cancelDeal()" if.bind="canCancel" type="tertiary">Cancel deal</pbutton>
+  <div class="dealActionButtons" if.bind="canEdit || canCancel">
```

Comment:
Split up "Share" from [ Edit+Cancel ]
by wrapping Edit+Cancel with its own `div`

When thinking about the problem, this emerged as the most fitting solution.
Edit+Cancel are variable, whereas "Share" is always present, thus makes sense to split up Edit+Cancel